timeline for project (#82)
sw-bachelor/sw6: report/sections/introduction/timeline-for-project.tex

This interview resulted in some of the prototypes getting reworked to better fit the customers needs.

\subsubsection{Sprint 2}

> As there was no release at the end of sprint 1 because of the decision to migrate to Flutter, many of the user stories assigned in Sprint 1 were not completed and due to this there was no reason to conduct a usability test in sprint 1.

Reviewer: ```suggestion
As there was no release at the end of sprint 1 due to the decision to migrate to Flutter, several of the user stories assigned in Sprint 1 were not completed and as a result of this, there was no reason to conduct a usability test in sprint 1.
```

> Instead sprint 2 was mostly spent developing user stories planned in sprint 1 and some extra user stories for sprint 2.

Reviewer: ```suggestion
Instead, sprint 2 was mostly spent developing user stories planned in sprint 1 and some additional user stories for sprint 2.
```

During this sprint the PO group also implemented some user stories, which are described in \autoref{our-assigned-user-stories-sprint-2}
At the end of this sprint the first release was completed.

\subsubsection{Sprint 3}
There were still some leftover user stories from sprint 1 and sprint 2, which were not completed due to some dependencies. 
At this point the project is starting to have a some of the most essential features implemented, but still a needing a lot of features before it being useable. 
This sprint focused mostly on implementing user stories.
However, there were also developed new prototypes during this sprint.

> This sprint ended with a usability test which are described in \todo{ref til rasmus' lort}

Reviewer: mere sigende todo pls

\subsubsection{Sprint 4}
As sprint 4 was our final sprint, the sprint mostly focused on finishing the work that was not completed in the previous sprints and developing the most important user stories to have a minimal viable product.
The PO group and process group did not implement user stories in this sprint, because they focused on documentation and making the handover to next year students as thorough as possible.
In the end of the sprint the final usability test was conducted.
The purpose of this usabiltiy test was to get a better understanding of what the application was lacking and what the next year students should focus on.

\subsubsection{Deadline}
All development on the weekplanner ended May 13th, after this all groups were expected to only focus on finishing their report.
The PO group still needed to finish some documentation for next year, as the last usability test was held on May 14th.

> Project handin was on May 28th.

Reviewer: ```suggestion
Project hand in was on May 28th.
```
